packages/terra-toggler/lib/Toggler.scss

```diff
@@ -0,0 +1,33 @@
+@import '~terra-mixins';
+
+// The accordion content panel doesn't have the header highlight, but the other ones do
+.terra-Toggler-header {
+  @include terra-text-align-start();
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  line-height: 1.5em;
+  padding: 0;
+  width: 100%;
+}
+
+.terra-Toggler-content {
+  overflow: hidden;
+}
+
+.terra-Toggler.is-collapsed > .terra-Toggler-content {
+  max-height: 0;
+}
+
+.terra-Toggler.is-expanded > .terra-Toggler-content {
+  max-height: 1000px;
+}
+
+.terra-Toggler.is-collapsed.is-animated > .terra-Toggler-content {
+  transition: max-height 0.25s ease-out;
+}
+
+.terra-Toggler.is-expanded.is-animated > .terra-Toggler-content {
+  transition: max-height 0.25s ease-in;
```

Comment:
out of curiosity, what do we value more, a consistent time to expand, or a consistent expand speed? If the content is super long, I think it will appear to open up faster than small content right?